packages/atomic/src/pages/examples/custom.html

```diff
@@ -11,7 +11,7 @@
     <script>
       (async () => {
         await customElements.whenDefined('atomic-search-interface');
-        const searchInterface = document.querySelector('#search');
```

Comment:
Is the `#search` id still needed? Or can it be removed from the html element?

Reply:
Not needed & it wasn't on the html element 😢